resolwe/test/utils.py

```diff
@@ -66,26 +71,83 @@ def check_docker():
     return available, reason
 
 
-def with_docker_executor(method):
-    """Decorate unit test to run processes with Docker executor."""
+def with_custom_executor(wrapped=None, **custom_executor_settings):
```

Comment:
It would be really nice if you implemented this as a subclass of Django's [TestContextDecorator](https://github.com/django/django/blob/master/django/test/utils.py#L320). Then we could use it to decorate the test classes or methods. Now only test methods can be decorated, right? See also the implementation of [override_settings](https://github.com/django/django/blob/master/django/test/utils.py#L385).